grpc-server/template_test.go

```diff
@@ -107,3 +110,76 @@ func TestCreateTemplate(t *testing.T) {
 		})
 	}
 }
+
+func TestGetTemplateByName(t *testing.T) {
+	type (
+		args struct {
+			db   mock.DB
+			name string
+		}
+		want struct {
+			expectedError bool
+		}
```

Comment:
can we make this something like:

```suggestion
		err bool
```
instead? No need to define a struct for just one type and its easier to use too.